MapleServer2/PacketHandlers/Game/SkillHandler.cs

```diff
@@ -53,14 +59,14 @@ public override void Handle(GameSession session, PacketReader packet)
 
         private void HandleFirstSent(GameSession session, PacketReader packet)
         {
-            long skillCount = packet.ReadLong();
```

Comment:
As stated above, keep these local variables. It makes no sense to set them elsewhere

Reply:
declaring it global in the class but private, give me the option to pass the value that needs from one method to another because is wip right now until skill are handle the correct way, what do you suggest? probably make another class to handle this variables to use it in the methods i need? for example, `skillId, skillLevel` is read in `HandleFirstSent` and i need them on `HandleAoeDamage` but is not being send on that packet.